### DISCUSSION TITLE
Translate Zustand docs to other language

### DISCUSSION THREAD

**Discussion by jangjia01234** [pmndrs/zustand#1894] [Ideas] [answered]
# Goals

- Translate the Zustand document into Korean.
- I want to make it an open source so that various Korean developers can contribute to this.
- Also, I would like to deploy the results and see what I've done.

# Background

As a user who can easily read or understand English documents, official document of Zustand is clearly and easily written. However, many Korean developers who are not familiar with English are having a hard time because its manual is only written in English. Recently, I discovered several times that some developers extract some of the document and make Korean translations because they need it while reading it.
And also, as a frontend developer who mainly uses React, I have seen that the official document of React are well translated into several languages. As a native Korean speaker, I was able to understand the document more quickly and proceed with the development based on this while reading the text translated into Korean. 
Therefore, just as the official React document supports several languages, I would like to start translating the official document of Zustand into Korean.

# Suggestions

I am currently contributing personally to [Next.js Official Document Korean Translation Work](https://github.com/Nextjs-kr/Nextjs.kr/issues/1).
Many Koreans are agreed and working hard for it to participate in translation. As a user who loves Zustand, I want to work on translation so that many Korean developers can understand Zustand better like this project.
I have already seen that the official document is only written in English and that no other languages are supported. Therefore, I am also considering making the seperated web page of Korean-translated version, such as [React official documents](https://ko.react.dev/). Is this also possible? I would appreciate it if you could check on the proposal and give me your opinion. ☺️

**Comment by dai-shi** [comment, marked as answer]
In the past, someone requested and we have ID translation in wiki: https://github.com/pmndrs/zustand/wiki/Translation-ID

I think what's better is you or anyone create a translation project and we put a link in docs (can be readme or maybe website).
We don't have a bandwidth to follow React official docs approach and distributed projects should work better as our community.

**Reply by jangjia01234** [reply]
Thanks a lot!🙌
I didn't know there was a document like this on the wiki. As you recommended, it seems better to make a Korean version separately and add a link. 
Since I've been thinking a lot about how to do so... it'll be helpful for me to proceed in this way! So, thank you for your quick and kind answer :)
